etc/listr2.api.md

```diff
@@ -245,17 +245,17 @@ export class ListrTaskObject<Ctx, Renderer extends ListrRendererFactory> extends
         count: number;
         withError?: any;
     };
-    run(context: Ctx, wrapper: ListrTaskWrapper<Ctx, Renderer>): Promise<void>;
-    skip: boolean | string | ((ctx: Ctx) => boolean | string | Promise<boolean> | Promise<string>);
+    run(context: Ctx | undefined, wrapper: ListrTaskWrapper<Ctx, Renderer>): Promise<void>;
+    skip: boolean | string | ((ctx: Ctx | undefined) => boolean | string | Promise<boolean> | Promise<string>);

```

Comment:
These could have been `undefined` all along, due to `strictNullChecks` not being enabled yet